region/crateria/west/Statues Room.json

```diff
@@ -213,6 +218,7 @@
         "f_TourianOpen",
         "canUnderwaterWalljump"
       ],
+      "flashSuitChecked": true,
```

Comment:
Without spinlock, insane sounds right.  It becomes a shot-try trick

Reply:
Do you think it's worth adding a helper to model this?

Reply:
practice hack has spinlock, so It's fine to add a helper for it.  That doesn't help with keyboard/hitbox.
But then it's hard to tell which uwu's are reasonable without spinlock.

Reply:
Makes sense. Maybe a spinlock helper isn't really necessary. Currently the randomizer always provides spinlock. And in other contexts where it might not be available, there's nothing stopping people from just disabling UWJ tech or treating it as higher difficulty